TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteFragment.kt

```diff
@@ -19,8 +17,16 @@ import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
 import com.microsoft.device.display.samples.twonote.model.DrawViewModel
+import java.io.*
```

Comment:
I could be wrong but I think ktlint doesn't like when imports have wildcards (but if you ran it and it already passed ktlint, no need to change)